### PR TITLE
Update EXM 130 module to be more relevant to the measure logic

### PR DESCRIPTION
**Pull request by mgramigna**
image

### CODE REVIEW

**Comment by tstrass** [comment]
The diffs are a little difficult to parse, can you summarize what was wrong and what changed?

**Comment by mgramigna** [comment]
Absurd request.

Yeah, the way the measure logic is, it looks for the colonoscopy 10 years or sooner before the start of the measurement period, whereas the encounter itself needs to happen during the measurement period. So we moved that numerator procedure state to be 10 years or sooner (assuming reporting year is 2019 i.e. 2009 onward). We then have a guard for the 2019 reporting year that transitions to the relevant encounter.

The way we wrote the module initially, it transitioned from the encounter to the procedure directly in the same year, but feedback from the breakout indicated that this isn't really in line with the measure logic, hence the changes here.

**Comment by tstrass** [comment]
Is this roughly the expected ratio? 
image

**Comment by mgramigna** [comment]
That's around what we were seeing, yeah

**Review by tstrass — APPROVED** [review]
LGTM
